test/suites/lib/skip.py

```diff
@@ -162,3 +162,4 @@ def skip_or_run_datetime_test(func):
 
     return skip_or_run_test_pcall_require(func, 'datetime',
                                       'does not support datetime type')
+
```

Comment:
I'm not sure it's necessary. There is already a newline at the end of file.

Reply:
Done